lib/driver.js

```diff
@@ -435,6 +442,43 @@ class AndroidDriver extends BaseDriver {
     }
   }
 
+  async setSharedPreferences () {
+    let sharedPrefs = this.opts.sharedPreferences;
```

Comment:
btw, let me know if you guys want to have the shared-preferences-builder repo inside appium. I can transfer the repo to you guys.